src/Leaflet.js

```diff
@@ -1,29 +1,170 @@
 
-var L = {
-	version: '1.0.3'
-};
+import {version} from '../package.json';
+export {version};
 
-function expose() {
-	var oldL = window.L;
+// control
 
-	L.noConflict = function () {
-		window.L = oldL;
-		return this;
-	};
+import {Control, control} from './control/Control';
+import {Layers, layers} from './control/Control.Layers';
+import {Zoom, zoom} from './control/Control.Zoom';
+import {Scale, scale} from './control/Control.Scale';
+import {Attribution, attribution} from './control/Control.Attribution.js';
 
-	window.L = L;
-}
+Control.Layers = Layers;
+Control.Zoom = Zoom;
+Control.Scale = Scale;
+Control.Attribution = Attribution;
 
-// define Leaflet for Node module pattern loaders, including Browserify
-if (typeof module === 'object' && typeof module.exports === 'object') {
-	module.exports = L;
+control.layers = layers;
+control.zoom = zoom;
+control.scale = scale;
+control.attribution = attribution;
 
-// define Leaflet as an AMD module
-} else if (typeof define === 'function' && define.amd) {
-	define(L);
-}
+export {Control, control};

```

Comment:
Adding a file `./control/index.js` and doing the preparation of `Control`/`control` there would streamline the code in `Leaflet.js` down to 
```javascript
export {Control, control} from './control';
```